sdk/keyvault/azure-security-keyvault-keys/inc/ApiViewSettings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "sourceFilesToProcess": null,
+  "additionalIncludeDirectories": [
+    "../../../core/azure-core/inc",
+    "../../azure-security-keyvault-shared/inc"
+  ],
+  "sourceFilesToSkip": [
+  ],
+  "additionalCompilerSwitches": [],
+  "includeInternal": false,
+  "includeDetail": false,
+  "includePrivate": false,
+  "filterNamespace": "Azure::Security::KeyVault::Keys",
+  "reviewName": "KeyVault Keys Review",
+  "serviceName": "Azure Security Keyvault Keys",
+  "packageName": "azure.security.keyvault.keys"
```

Comment:
Where are these package names coming from? I don't think we use the dot separated name anywhere.
Should these match what is posted on vcpkg, instead?
For example:
`azure-security-keyvault-keys-cpp`

https://vcpkg.io/en/packages.html
> .\vcpkg install azure-security-keyvault-keys-cpp

Even the docs are hyphen separated:
https://azuresdkdocs.blob.core.windows.net/$web/cpp/azure-security-keyvault-keys/4.3.0/index.html

Reply:
THe package name should match the public package name. I made all of these up in a flurry in one afternoon, they may be wrong for some packages.

I've asked the package owners to please correct the ones which are incorrect, I've gotten some corrections already.

Reply:
I see. What are these package name strings used for? For example, would having the suffix `-cpp` be an issue?

This raises an interesting question, what should we consider as our package names? Is it what we publish to vcpkg, verbatim?